src/openforms/formio/api/urls.py

```diff
@@ -10,4 +10,9 @@
         TemporaryFileUploadView.as_view(),
         name="temporary-file-upload",
     ),
+    path(
+        "mapsearch",
+        MapSearchView.as_view(),
+        name="map-search",
+    ),
```

Comment:
this functionality has nothing to do with Form.io (it just happens to be used in a custom form.io component), so I'd move this to a different place in the codebase.

IMO `src/openforms/contrib/kadaster` is a good candidate - you can implement the `api` package there and wire it up in the main URL routes under `/api/v2/geo/address-search`.

I want to "mask" the fact that we use PDOK/Kadaster APIs for this a bit (it's not a secret), but structuring the API and code this way makes it easier to plug in alternative backends if that need ever becomes relevant.